assets/js/util/tracking/index.js

```diff
@@ -23,7 +23,7 @@ import createTracking from './createTracking';
 
 const {
 	isFirstAdmin,
-	trackingAllowed,
+	isSiteKitScreen,
```

Comment:
This `isSiteKitScreen` variable should be also added to the `initialConfig` object.

Reply:
https://github.com/google/site-kit-wp/pull/4142/commits/843898435e56fcc390db600c7cbdfa99f3b02075